src/umpire/strategy/MixedPool.hpp

```diff
@@ -0,0 +1,62 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory
+//
+// Created by David Beckingsale, david@llnl.gov
+// LLNL-CODE-747640
+//
+// All rights reserved.
+//
+// This file is part of Umpire.
+//
+// For details, see https://github.com/LLNL/Umpire
+// Please also see the LICENSE file for MIT license.
+//////////////////////////////////////////////////////////////////////////////
+#ifndef UMPIRE_MixedPool_HPP
+#define UMPIRE_MixedPool_HPP
+
+#include <memory>
+#include <vector>
+#include <functional>
+
+#include "umpire/strategy/AllocationStrategy.hpp"
+
+#include "umpire/strategy/DynamicPool.hpp"
+#include "umpire/strategy/FixedPool.hpp"
+
+#include "umpire/Allocator.hpp"
+
+namespace umpire {
+namespace strategy {
+
+class MixedPool :
+  public AllocationStrategy
+{
+  public:
+    MixedPool(
```

Comment:
@davidbeckingsale, shouldn't this take the same default arguments as DynamicPool so that it knows how to construct the underlying DynamicPool object for larger allocations? ([min_initial_alloc_size, min_alloc_size, and heuristic_fun](https://github.com/LLNL/Umpire/blob/feature/UM-281-faster-pool/src/umpire/strategy/DynamicPool.hpp#L69))

Reply:
Good suggestion. I will add more comprehensive Doxygen to the class too.